Lib/test/test_gzip.py

```diff
@@ -467,6 +467,25 @@ def sizes():
             self.assertEqual(f.read(100), b'')
             self.assertEqual(nread, len(uncompressed))
 
+    def test_no_refloop_with_writestream(self):
+        """Test that the _WriteBufferStream __del__ method prevents reference loops."""
```

Comment:
Could you link the issue here? Something like:
```suggestion
        """GH-129640: Test that the _WriteBufferStream __del__ method prevents reference loops."""
```